packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -344,6 +344,8 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
             required={required}
             htmlFor={id + '-input'}
             className={this._classNames.label}
+            aria-level={2}
+            role="heading"
```

Comment:
I doubt we'd want headings built in to combo boxes or other simple controls. Headings are mean to provide hierarchical  page structure.